pkgs/development/python-modules/pysam/default.nix

```
@@ -27,17 +28,49 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [ samtools ];
-  buildInputs = [ bzip2 curl cython lzma zlib ];
+  buildInputs = [
+    bzip2
+    curl
+    cython
+    libdeflate
+    lzma
+    zlib
+  ];
 
   checkInputs = [ pytest bcftools htslib ];
-  checkPhase = "py.test";
 
-  # tests require samtools<=1.9
-  doCheck = false;
-  preCheck = ''
+  # # tests require samtools<=1.9
+  # doCheck = false;
+
+  checkPhase = ''
```

pytestCheckHook would be much easier to use
```suggestion
  preCheck = ''
```